thunderfish/chirp.py

```diff
@@ -153,14 +153,14 @@ def chirp_analysis(data, samplerate, cfg):
     # '2016_04_27__downstream_stonewall_at_pool' made in colombia, 2016.
     ###
     import sys
-    import dataloader as dl
-    import config_tools as ct
+    from dataloader import load_data
+    from config_tools import get_config_dict
```

Comment:
why not "from .config_tools import get_config_dict" in this case ?
is the dot not required here ? and if so, why ?